fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -224,6 +227,21 @@ def run
         end
       end
 
+      command :env do |c|
+        c.syntax = 'fastlane env'
+        c.description = 'Print your fastlane environment, use this when you submit an issue on GitHub'
+        c.action do |args, options|
+          require "fastlane/environment_printer"
+          env_info = Fastlane::EnvironmentPrinter.get
+          puts env_info
+          if UI.confirm("🙄  Wow, that's a lot of markdown text... should fastlane put it into your clipboard, so you can easily paste it on GitHub?")
```

Comment:
`pbcopy` will only work for Mac users, so should we avoid showing it to people on other platforms?